### DISCUSSION TITLE
Error after latest update HA 2023.8.0 & OS 10.4

### DISCUSSION THREAD

**Discussion by Sawadee2u** [HASwitchPlate/HASPone#237] [Q&A]
Just did an update to:
Home Assistant 2023.8.0  <———-
Supervisor 2023.07.1
Operating System 10.4 <————
Frontend 20230802.0 - latest

Now getting this warning:

Some MQTT entities have an entity name that starts with the device name. This is not expected. To avoid a duplicate name the device name prefix is stripped of the entity name as a work-a-round. Please inform the maintainer of the software application that supplies the affected entities to fix this issue.

List of affected entities:

light.haspone_2_unselected_foreground_color
light.haspone_1_selected_background_color
number.haspone_1_active_page
sensor.haspone_1_sensor
light.haspone_1_unselected_background_color
light.haspone_3_unselected_foreground_color
light.haspone_3_selected_foreground_color
light.haspone_1_unselected_foreground_color
light.haspone_1_selected_foreground_color
light.haspone_2_backlight
sensor.haspone_3_sensor
sensor.witty_1_65_status
light.haspone_2_selected_background_color
number.haspone_3_active_page
number.haspone_2_active_page
light.haspone_3_unselected_background_color
light.haspone_2_selected_foreground_color
light.haspone_3_selected_background_color
light.haspone_1_backlight
sensor.haspone_2_sensor
light.haspone_2_unselected_background_color
light.haspone_3_backlight

This will stop functioning with 2024.2.0

What to do?

**Comment by aderusha** [comment]
Right now, no changes are needed, the message can be safely ignored, and we'll have 6 months to sort this out.  Annoyingly, this is a breaking change which will require blueprint and firmware updates.  More annoyingly, HA has no good means of updating blueprints so we'll need to figure out something for that too.

Tracking this issue here: https://github.com/HASwitchPlate/HASPone/issues/236

HA devs decided to drop this breaking change on us not by noting it in the breaking changes, but by throwing errors all of a sudden in everyone's installation.  See forum thread here: https://community.home-assistant.io/t/psa-mqtt-name-changes-in-2023-8/598099
